pkg/scalers/aws_dynamodb_scaler.go

```diff
@@ -5,16 +5,16 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"strconv"
+
 	"github.com/aws/aws-sdk-go-v2/aws"
 	"github.com/aws/aws-sdk-go-v2/service/dynamodb"
 	"github.com/aws/aws-sdk-go-v2/service/dynamodb/types"
 	"github.com/go-logr/logr"
+	kedautil "github.com/kedacore/keda/v2/pkg/util"
```

Comment:
This will fail, We have a static rule to organize the imports in 3 groups, runtime, deps, internals. this internal import has to be in another split group